config/content.d/developer.rackspace.com.json

```diff
@@ -31,11 +31,11 @@
           "/docs/cloud-feeds/v1/developer-guide/": "https://github.com/rackerlabs/standard-usage-schemas",
           "/docs/dedicated-load-balancers/v2/": "https://github.com/rackerlabs/docs-dedicated-networking/",
           "/docs/glossary/": "https://github.com/rackerlabs/docs-rackspace/",
-          "/docs/private-cloud/release-notes/v13": "https://github.com/rcbops/rpc-openstack/v13/",
+          "/docs/private-cloud/rpc/v14/": "https://github.com/rackerlabs/docs-rpc/v14/",
           "/docs/private-cloud/rpc/v13/": "https://github.com/rackerlabs/docs-rpc/v13/",
           "/docs/private-cloud/rpc/v12/": "https://github.com/rackerlabs/docs-rpc/v12/",
```

Comment:
You might want to put redirects in place for `/docs/private-cloud/rpc/v10` and `/docs/private-cloud/rpc/v11` if you don't already have them up.